README.md

```diff
@@ -1,21 +1,19 @@
-# SDE Console
+# sdx-console
 
-The Survey Data Exchange (SDE) Console is a component of the Office of National Statistics (ONS) SDE project, which takes an encrypted json payload and transforms it into a number of formats for use within the ONS. This console allows for insertion of input and checking of transformed output files, using the perkin and posie services.
+[![Build Status](https://travis-ci.org/ONSdigital/sdx-console.svg?branch=master)](https://travis-ci.org/ONSdigital/sdx-console)
 
-Installation
+The Survey Data Exchange (SDX) Console is a component of the Office of National Statistics (ONS) SDX project, which takes an encrypted json payload and transforms it into a number of formats for use within the ONS. This console allows for insertion of input and checking of transformed output files, using the sdx-decrypt and sdx-validate services.
 
-Using virtualenv and pip, create a new environment and install within using:
+## Usage
 
-	$ pip install -r requirements.txt
+To use the console you will need to start all services.
 
-It's also possible to install within a container using docker. From the sde-console directory:
+To get the environment running:
 
-	$ docker build -t sde-console .
+  - ./init.sh  # clones submodules
+  - ./update.sh  # pulls the latest version of each submodule, runs `mvn package` in the `perkin` project and calls `docker-compose up`
 
-Usage
+After a `docker-compose up`, the `sdx-console` app will be exposed on the host ip address (on port 80).
 
-To start the console, just run the server, which exposes a server on port 5000:
+SDE console has two endpoints: the default '/', which takes JSON as input, encrypts it and places it upon the app queue for SDE to decrypt and transform. The '/decrypt' endpoint takes an encrypted payload as input and decrypts to json and transforms to final formats.
```

Comment:
I think it would be clearer to list the API endpoints like in the [sdx-sequence](https://github.com/ONSdigital/sdx-sequence) repo